Cargo.lock

```diff
@@ -487,7 +487,7 @@ dependencies = [
  "env_logger 0.5.3 (registry+https://github.com/rust-lang/crates.io-index)",
  "habitat_api_client 0.0.0",
  "habitat_common 0.0.0",
- "habitat_core 0.0.0 (git+https://github.com/habitat-sh/core.git)",
```

Comment:
I'm wondering if this change was due to temporarily overriding the source of `core`? Is it possible this isn't intended?

Reply:
yeah this definitely looks like a dev hack not meant for the world.